### PR TITLE
Release v0.3.0 

### PR DESCRIPTION
**Pull request by erpankajpatel**
- Signature change for initialize method.
- Performance improvement for better loader and load times.
- Internal fixes.

### CODE REVIEW

**Review by chiragphyllo — APPROVED** [review]

File: android/src/main/java/com/phylloconnect/PhylloConnectModule.kt [hunk 1]
```diff
@@ -19,7 +19,9 @@ import android.view.View
 import android.content.Intent
 import com.facebook.react.bridge.WritableArray
 import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.ReadableMap
 import com.facebook.react.modules.core.DeviceEventManagerModule
+import kotlin.math.log
```
Comment (akshaybhalotia) [inline android/src/main/java/com/phylloconnect/PhylloConnectModule.kt:24]: Why is `math.log` required?

File: example/android/app/build.gradle [hunk 2]
```diff
@@ -175,7 +169,7 @@ android {
             //signingConfig signingConfigs.debug
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-            signingConfig signingConfigs.release
+            signingConfig signingConfigs.debug
```
Comment (akshaybhalotia) [inline example/android/app/build.gradle:172]: Shouldn't this be in release mode?

File: example/package.json [hunk 3]
```diff
@@ -19,8 +19,8 @@
     "base-64": "^1.0.0",
     "react": "^17.0.2",
     "react-native": "^0.67.1",
-    "react-native-bouncy-checkbox": "^2.1.10",
-    "react-native-phyllo-connect": "^0.2.0",
+    "react-native-bouncy-checkbox": "^3.0.2",
+    "react-native-phyllo-connect": "file:../react-native-phyllo-connect-0.3.0.tgz",
```
Comment (akshaybhalotia) [inline example/package.json:23]: In release code, change the plugin version to the published version

File: src/PhylloConnect.ts [hunk 4]
```diff
@@ -14,6 +14,7 @@ interface IPhylloInitialize {
   userId: string
   environment: PhylloEnvironment
   workPlatformId?: string
+  singleAccount?: Boolean
```
Comment (akshaybhalotia) [inline src/PhylloConnect.ts:17]: Why is it exposed?

**Review by akshaybhalotia — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/phylloconnect/PhylloConnectModule.kt (hunk 1 above)
Comment (erpankajpatel) [reply android/src/main/java/com/phylloconnect/PhylloConnectModule.kt:24]: hmm i don't think so its using in kotlin

Comment (akshaybhalotia) [reply android/src/main/java/com/phylloconnect/PhylloConnectModule.kt:24]: Then can we remove it?

Comment (erpankajpatel) [reply android/src/main/java/com/phylloconnect/PhylloConnectModule.kt]: @akshaybhalotia i have removed this one .

File: example/android/app/build.gradle (hunk 2 above)
Comment (erpankajpatel) [reply example/android/app/build.gradle:172]: this is to make debug apk
releas apk mostly use to upload for playstore , developer can make  change as per mobile app requirement

File: example/package.json (hunk 3 above)
Comment (erpankajpatel) [reply example/package.json]: yes now we are going publish , we can change it , i have changed

File: src/PhylloConnect.ts (hunk 4 above)
Comment (chiragphyllo) [reply src/PhylloConnect.ts:17]: Akshay, this is in the React native package code, not in the example folder.

**Review by akshaybhalotia — APPROVED** [review]

**Review by chiragphyllo — APPROVED** [review]

**Pull request merged**